client/src/plus/deep-dives/overview/app.tsx

```diff
@@ -1,107 +1,266 @@
-        <div className="header-wrapper">
-          <div className="header-content">
-            <div className="heading-group">
-              <h1>Modern CSS in the Real World</h1>
-              <h2>Featured Deep Dive</h2>
-            </div>
-            <p className="author">Rachel Andrew, CSS WG Invited Expert</p>
-            <ul className="author-links">
-              <li>
-                <a href="https://twitter.com/rachelandrew" rel="external">
-                  @rachelandrew
-                </a>
-                ,
-              </li>
-              <li>
-                <a href="https://rachelandrew.co.uk" rel="external">
-                  rachelandrew.co.uk
-                </a>
-              </li>
-            </ul>
-          </div>
+        <div className="header-content">
+          <h1>Modern CSS in the Real World</h1>
+          <p className="author">
+            <a href="https://twitter.com/rachelandrew" rel="external">
+              @rachelandrew
+            </a>
+          </p>
         </div>
       </header>
       <div className="overview-content-container">
-        <h3>About</h3>
         <p>
           One thing is clear about the web - the technology moves fast. There
           are new features and tools coming out all the time, and a pressure to
-          try to keep up with the latest and greatest. This is often juxtaposed
-          with the frustration of wanting to use those new features but finding
-          it hard to do so while still supporting the browser set that your user
-          base still uses.
-        </p>
-        <p>
-          This is where our "Practical web compatibility" series comes in
-          &mdash; expert knowledge distilled into easy-to-follow articles that
-          provide practical solutions for using modern tech while not leaving
-          older browsers behind.
-        </p>
-        <p>
-          "Modern CSS in the Real World" begins by applying this formula to CSS.
-          We will be covering the tools at your disposal to cope with browser
-          compatibility, from those which help you identify bugs and find out
-          what is supported where, to those which help create fallbacks, or
-          polyfill support. We’ll be working through example patterns to see how
-          we can create a good experience, despite different browser
-          capabilities.
-        </p>
-        <p>
-          This first article will provide you with a high-level summary of the
-          types of CSS issues you’ll be facing, and a good framework for
-          deciding on a support strategy, and communicating it with your team.
-        </p>
-        <p>
-          Subsequent articles will look in-depth at the tools and techniques
-          you’ll be using to solve such issues, and present reusable patterns
-          that demonstrate those tools and techniques in practice.
+          try to keep up with the latest and greatest.{" "}
```

Comment:
Would it be possible to have an ellipsis before the text ahead of "see more" like so: 
![image](https://user-images.githubusercontent.com/44966503/129592797-de5fb414-080c-4618-b495-d3a5293afe16.png)